src/plugins/data_views/server/rest_api_routes/internal/fields_for.ts

```diff
@@ -69,6 +70,11 @@ export const querySchema = schema.object({
   include_unmapped: schema.maybe(schema.boolean()),
   fields: schema.maybe(schema.oneOf([schema.string(), schema.arrayOf(schema.string())])),
   allow_hidden: schema.maybe(schema.boolean()),
+  field_types: schema.maybe(
+    schema.oneOf([schema.string(), schema.arrayOf(schema.string())], {
```

Comment:
Do we still want to allow a single string here if types are defined as `field_types?: string[];`?

Reply:
It seems so. I'm verifying that a single value is supported and there's an integration test to go along with it. The test passes as is, but if I remove `schema.string()` then the test fails.

Reply:
Okay, then we should keep the typescript types and the schema aligned for `field_types`, right?  Returning `field_types?: string | string[];` would address it.

Reply:
Same for `fields` btw.